src/actions/index.js

```diff
@@ -49,18 +42,39 @@ export function getRemoteManifest(url) {
   };
 }
 
-// Player related
+// Nav actions
+export function navItemClick(url) {
+  return {
+    type: types.NAV_ITEM_CLICK,
+    payload: url
+  };
+}
+
+export function setStartTime(time) {
+  return {
+    type: types.NAV_STARTTIME,
+    payload: time
+  };
+}
+
+export function setUnclick() {
```

Comment:
I think `resetClick` and `types.NAV_RESET_CLICK` would be better names for these.  What do you think?

Reply:
Yes that sounds better that this :+1: